app/javascript/dashboard/modules/widget-preview/components/Widget.vue

```diff
@@ -32,52 +63,202 @@ export default {
+          checked: false,
+        },
+      ],
+      isDefaultScreen: true,
+      isWidgetVisible: true,
+    };
   },
   computed: {
     getWidgetHeadConfig() {
       return {
         welcomeHeading: this.welcomeHeading,
-        welcomeTagLine: this.welcomeTagLine,
+        welcomeTagline: this.welcomeTagline,
         websiteName: this.websiteName,
-        websiteDomain: this.websiteDomain,
         logo: this.logo,
-        isExpanded: this.isExpanded,
+        isDefaultScreen: this.isDefaultScreen,
+        isOnline: this.isOnline,
+        replyTime: this.replyTimeText,
+        color: this.color,
+      };
+    },
+    getWidgetBodyConfig() {
+      return {
+        welcomeHeading: this.welcomeHeading,
+        welcomeTagline: this.welcomeTagline,
+        isDefaultScreen: this.isDefaultScreen,
         isOnline: this.isOnline,
-        replyTime: this.replyTime,
+        replyTime: this.replyTimeText,
+        color: this.color,
+        logo: this.logo,
       };
     },
+    getWidgetFooterConfig() {
+      return {
+        isDefaultScreen: this.isDefaultScreen,
+        color: this.color,
+      };
+    },
+    replyTimeText() {
+      switch (this.replyTime) {
+        case 'in_a_few_minutes':
+          return this.$t(
+            'INBOX_MGMT.WIDGET_BUILDER.REPLY_TIME.IN_A_FEW_MINUTES'
+          );
+        case 'in_a_day':
+          return this.$t('INBOX_MGMT.WIDGET_BUILDER.REPLY_TIME.IN_A_DAY');
+        default:
+          return this.$t('INBOX_MGMT.WIDGET_BUILDER.REPLY_TIME.IN_A_FEW_HOURS');
+      }
+    },
+    getBubblePositionStyle() {
+      return {
+        justifyContent: this.widgetBubblePosition === 'left' ? 'start' : 'end',
+      };
+    },
+    getBubbleTypeClass() {
+      return {
+        'bubble-close': this.isWidgetVisible,
+        'bubble-expanded':
+          !this.isWidgetVisible && this.widgetBubbleType === 'expanded_bubble',
+      };
+    },
+    getWidgetBubbleLauncherTitle() {
+      return this.isWidgetVisible || this.widgetBubbleType === 'standard'
+        ? ' '
+        : this.widgetBubbleLauncherTitle;
+    },
+  },
+  methods: {
+    handleScreenChange(item) {
+      this.isDefaultScreen = item.id === 'default';
+    },
+    toggleWidget() {
+      this.isWidgetVisible = !this.isWidgetVisible;
+      this.isDefaultScreen = true;
+    },
   },
 };
 </script>
 
 <style lang="scss" scoped>
-.text-lg {
-  font-size: var(--font-size-default);
+.screen-selector {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
 .widget-wrapper {
-  box-shadow: var(--shadow-larger);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  box-shadow: 0 0px 20px 5px rgb(0 0 0 / 10%);
   border-radius: var(--border-radius-large);
-  background-color: var(--color-background);
-  z-index: 99;
+  background-color: #f4f6fb;
+  min-width: calc(var(--space-large) * 10);
+  max-width: calc(var(--space-large) * 10);
```

Comment:
@aswindevps Insted of using min and max with same, i think we can use width.

Reply:
Make sense

Reply:
Done